src/main/java/com/revature/models/Admin.java

```diff
@@ -24,6 +30,7 @@
 @Component
 @Entity
 @Table(name="admins")
+@Getter @NoArgsConstructor @Setter @EqualsAndHashCode @ToString
```

Comment:
All of these (except NoArgsConstructor), can actually be replaced with an @ Data annotation. It is being used in some other places as well, so it would be nice to have that consistency.